web-console/future-openshift/custom-catalogs/custom-catalogs.md

```diff
@@ -0,0 +1,65 @@
+# Adding custom catalogs to OperatorHub
+
+Admins can create custom catalogs in order to give admin console users the ability to add operators from other sources, beyond the default OperatorHub catalog sources.
+
+## Global Configuration
+
+OperatorHub sources are found through: Administation > cluster settings > global configuration > OperatorHub / sources tab.
+
+![global config](img/operatorhub-custom-catalog1.png)
+
+![OperatorHub config](img/operatorhub-custom-catalog2.png)
+
+## Sources
+
+In the OperatorHub configuration, users can see the current default sources that are feeding into OperatorHub. These sources populate OperatorHub and keep installed operators updated.
+
+![OperatorHub sources default](img/operatorhub-custom-catalog3.png)
+
+If users "create catalog source" they will see a form to fill out details.
+
+![OperatorHub catalog source form](img/operatorhub-custom-catalog4.png)
+
+After the form is filled out and users select "create", they will see the newly created catalog source in the list.
+
+![OperatorHub catalog source in list](img/operatorhub-custom-catalog5.png)
+
+Once a new catalog source is created, admin users can see the operators from the catalog source from within OperatorHub.
+
+![OperatorHub with custom catalogs](img/operatorhub-custom-catalog6.png)
+
+
+## Actions
+
+Custom catalog sources can be edited or deleted from the kebab menu in the list.
+
+![edit catalog source](img/operatorhub-custom-catalog7.png)
+
+If users delete a custom catalog source, they must first confirm the deletion by typing the catalog source name. The deletion confirmation step will also inform users that if a source is deleted, any operators already installed from the source will no longer receive updates.
+
+![delete catalog source confirm](img/operatorhub-custom-catalog8.png)
```

Comment:
I noticed in a [recent PR](https://github.com/openshift/openshift-origin-design/pull/265) from @bmignano around bulk actions, a similar delete modal asked the user to simply enter "delete" to confirm. Is this something we should standardize on? Maybe we always use 'delete'? Or maybe bulk action is just a special case because we can't rely on having one resource name for the user to enter :)

Reply:
I'll see what the team thinks here in the design channel – will update this if we agree this should be the new convention

Reply:
Updated to show "delete" rather than [name of custom catalog] for typing when deleting.